termenv_unix.go

```diff
@@ -113,6 +113,79 @@ func (o Output) backgroundColor() Color {
 	return ANSIColor(0)
 }
 
+func (o Output) kittyKeyboardProtocolSupport() byte {
```

Comment:
We probably want to combine both this and `termStatusReport` into one routine that gets executed when the output is being initialized. Right now, both `kittyKeyboardProtocolSupport` & `termStatusReport` will set the tty termios and read the envs before reading the response. This results in multiple unnecessary calls to `GetTermios/SetTermios` and `Getenv`.

Instead, we could define an `initialize` function that queries the terminal once. However, I believe this might break things since we query the terminal during the lifetime of `WithColorCache` https://github.com/muesli/termenv/pull/159/files#diff-35d16970a063a59d666ecdce46f6ac3363f36d17a3e672d8ff69f9ef2056d4cbR114

cc/ @muesli